Update support.md

Jarli01 opened the pull request:
Updated verbiage a bit more.

### Check list

> Check items when done or if not relevant

- [ ] PR reference the relevant issue (e.g. `Fixes #007`)
- [ ] if UI changes, a screenshot has been added to the PR
- [ ] `CHANGELOG.unreleased.md`:
   - enhancement/bug fix entry added
   - list of packages to release updated (`${name} v${new version}`)
- [x] documentation updated
- [ ] **I have tested added/updated features** (and impacted code)

### Process

1. create a PR as soon as possible
1. mark it as `WiP:` (Work in Progress) if not ready to be merged
1. when you want a review, add a reviewer
1. if necessary, update your PR, and re- add a reviewer

Jarli01 committed:
Update support.md

Updated verbiage a bit more.

olivierlambert commented on docs/support.md:
```diff
@@ -19,9 +19,9 @@ If you are using Xen Orchestra via the sources, you can ask questions and try to
+If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.   
 
-If you are using Xen Orchestra in production, please subscribe to a plan.
+Lastly while Xen Orchestra is free to use and change, supporting and developing it takes a lot of effort from a group of talented people, if you consider support business critical please sign-up for one of our [support plans](https://xen-orchestra.com/#!/xo-pricing). 
```
We won't call ourselves like this :wink: Also we encourage people using in production to take support, not just only "business critical" missions.

Jarli01 replied:
Won't call ourselves _like what_?

The deciding factor for any business to implement a solution should include "do I require support for this" if so, support is business critical and if support doesn't exist then the product is a Hobbyist solution at best. Since XOA is the solution that with a paid plan includes support this verbiage makes more sense.

olivierlambert replied:
1. Like "talented people".
2. "Critical" means everything outside "critical" is OK from the sources. So it's like "critical" is even more a niche than "any" production usage. That's why I think this wording is not fit: every production load (ie in a company) makes sense. Hobbyist is clearly not related to "production", that's why "critical" word isn't needed here

Jarli01 replied:
So you'd rather the PR say that the dev team of VatesFR isn't talented?

As for 2, using the word critical is anything that is a production work load, and specifically highlights in the English language that if:

**You don't have a support contract, then clearly you are OKAY with possibly severe outages and issues and that you accept these risks, and that you are willing to either fix the issue yourself or depend on the kindness of others.**

olivierlambert replied:
1. I would like to avoid telling anything regarding our talent or not in the official documentation, because it's a purely subjective view
2. I don't know why we should insist about telling that, the initial wording seems pretty clear: in production (aka "real" env), please take support. Short, clear and concise. I don't understand why we should insist otherwise, as a hobbyist you know you don't run into production. As a company, it won't be serious to take a backup product without taking support or being able to debug the software.

olivierlambert replied:
FYI, the rest of the PR is OK for me.

Jarli01 replied:
IDK what you want for point 1, seems a bit insane to not speak positively about the people that contribute to this project. 

For number 2 that's just it though, you as the owner and supplier of support need to classify these installations "source installations" as hobbyist if they aren't paying for support so you have a way to separate the issues. 

The end user/admin may not do so, but you have to so you have a means of qualifying an installation as "supportable".

olivierlambert replied:
1. Thing is: this is XO official documentation, we'd like to keep it neutral.
2. I don't think this require more clarification: you can clone something from GitHub, and there is a bug tracker at the very same place you clone it (if you want to do so), as any other OSS project on GH. Keep in mind this documentation is also used by XOA users, it's not targeting specifically "source users" (aka not XOA users). We don't have any issue "classifying" is GH tickets were made by hobbyist: all pro support comes into our internal support system. Our main issue was to debug issues coming from various platforms and installers, but this will be addressed in the future :)

Anyway, the goal of the original sentence was to just remind people using XO in production to use a validated solution (XOA) with support. Nothing more is needed IMHO.

Jarli01 committed:
Update support.md

Critical just makes sense to an English first reader. Not including it means that "everything and anything may or may not be critical". 

Therefore I'm leaving it in there. If you'd like to decline the PR, fine. 

Arguing with you over the sentence structure of an English targeted product isn't worth my time.

olivierlambert commented on docs/support.md:
```diff
@@ -19,9 +19,9 @@ If you are using Xen Orchestra via the sources, you can ask questions and try to
 * In our IRC - `#xen-orchestra` on `Freenode`
 
-However, there's no guarantee you will receive an answer and no guaranteed response time. If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.  
+If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.   
```
Is it an extra space after the end of the sentence? If yes, please remove it :)

olivierlambert replied:
Is this also an extra space here?

olivierlambert commented on docs/support.md:
```diff
@@ -6,11 +6,11 @@ You can access our pro support if you subscribe to any of these plans:
 * Premium
 
-The better the plan, the faster the support will be with higher priority.
+Higher tier support plans include faster ticket response times. Paid support plans and response times are based on the plan you have, plans can be [reviewed here](https://xen-orchestra.com/#!/xo-pricing). 
```
Extra space after the end of the sentence?

olivierlambert commented on docs/support.md:
```diff
@@ -19,9 +19,9 @@ If you are using Xen Orchestra via the sources, you can ask questions and try to
+If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.   
 
-If you are using Xen Orchestra in production, please subscribe to a plan.
+Lastly while Xen Orchestra is free to use and change, supporting and developing it takes a lot of effort. If you consider support business critical please sign-up for one of our [support plans](https://xen-orchestra.com/#!/xo-pricing). 
```
Idem (but I'm not sure it's a space?)

Jarli01 committed:
Update support.md

All extraneous spaces removed.

Jarli01 committed:
Update support.md

Carriage return - removed.

Jarli01 committed:
Update support.md

Space removed.

olivierlambert commented on docs/support.md:
```diff
@@ -19,9 +19,9 @@ If you are using Xen Orchestra via the sources, you can ask questions and try to
+If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.  
 
-If you are using Xen Orchestra in production, please subscribe to a plan.
+Lastly while Xen Orchestra is free to use and change, supporting and developing it takes a lot of effort. If you consider support business critical please sign-up for one of our [support plans](https://xen-orchestra.com/#!/xo-pricing).
```
If you want more details than the original sentence, what do you think about:

"If you are using Xen Orchestra in production, please subscribe to a plan to get turnkey XO virtual Appliance (XOA) with QA, web updater and pro support. Xen Orchestra itself is Open Source, but supporting, developing  and maintaining it takes a lot of effort."

Jarli01 replied:
That sounds a little pushy to get people to purchase the XOA. And while I agree, production use should mandate that a business need support, I can't make that justification for every business. 

A not-for-profit charity may want to use the solution but be scared away from the product by seeing wording like that.

Jarli01 replied:
Proposed changes have been made

olivierlambert commented on docs/support.md:
```diff
@@ -6,11 +6,11 @@ You can access our pro support if you subscribe to any of these plans:
 ## Free limited support
 
-In the free edition of Xen Orchestra, you also have ability to open support ticket. We will do our best to assist you, however, this support is limited and has no guaranteed response time. 
+With the free edition of Xen Orchestra, you can open support tickets and we will do our best to assist you, however, this support is limited and is not guaranteed in regards to response times or resolutions offered.
```
Maybe we should make a clear distinction between XOA and "the sources". So in the previous paragraph, explaining that pro support comes with XOA as a turnkey/validated platform, and then explain that even XOA Free got pro support on the same platform but without response time guarantee. Then the next title/paragraph is a good transition toward "the sources" and it seems logical. What do you think?

olivierlambert commented on docs/support.md:
```diff
@@ -19,9 +19,9 @@ If you are using Xen Orchestra via the sources, you can ask questions and try to
 * In our IRC - `#xen-orchestra` on `Freenode`
 
-However, there's no guarantee you will receive an answer and no guaranteed response time. If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.  
+If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.  
```
It seems this sentence is almost the same than previous the bullet list, maybe we should "group" this.

Jarli01 replied:
If you are using Xen Orchestra via the source and not XOA, you can ask questions and try to recieve help a number of different ways:

* In our [forum](https://xcp-ng.org/forum/category/12/xen-orchestra)
* In our IRC - `#xen-orchestra` on `Freenode`

We encourage you to give back to the community by assisting other users via these two avenues as well.

Jarli01 committed:
Update support.md

Jarli01 committed:
Update support.md

Added "through" which I meant to add

Jarli01 committed:
Update support.md

olivierlambert commented on docs/support.md:
```diff
@@ -2,26 +2,27 @@
+We encourage you to give back to the community by assisting other users via these two avenues as well.
 
-If you are using Xen Orchestra in production, please subscribe to a plan.
+Lastly while Xen Orchestra is free and Open Source software, supporting and developing it takes a lot of effort. If you consider professional support business critical, please sign-up for one of our [support plans](https://xen-orchestra.com/#!/xo-pricing).
```
LGTM except this piece: again, "business critical" is IMHO removing value of support for "regular" company usage. It's not neutral and push pro support in a niche corner. Pro support of a backup solution should be the norm, not the exception. The way I understand your wording is the opposite.

I'm fine to keep it "as it was" if we can't find a consensus on this sentence.

Jarli01 replied:
@olivierlambert the way it reads is, if you consider the data in your environment critical, please sign-up for a paid plan. 

This is the original: "If you are using Xen Orchestra in production, please subscribe to a plan"

I have a production lab in my house, but I would never consider needing professional support by the Vates team for my home environment. 

That is why I think a distinction is needed here. 

Maybe something like this would work:

If you consider professional support a necessity, please sign up for one of our [support plans.](https://xen-orchestra.com/#!/xo-pricing)

Jarli01 replied:
And the reason that the verbiage I laid out is so specific on the business critical aspect is because, as a business, my servers are my means of making money. So spending for support there makes sense. 

But in a case of a home lab, that is running production systems for home use, there is no money being made there, and asking/pushing someone onto a paid platform there makes no sense.

Jarli01 replied:
An alternative that could be made is 

``` ... If you need professional support, we have several support options [available](https://xen-orchestra.com/#!/xo-pricing). ```

olivierlambert replied:
We don't have the same opinion on the "production" word ;) However, it's less risky to use "production". Why? Because we had literally nobody (since XOA started) paying for XOA for a home usage. We don't even sell to individuals (and we don't want to). On the other hand, some people could consider "critical business" outside their perimeter despite the fact it's a company.

To recap, there's zero (literally) chances to make people buy it by mistake, however "business critical" could reduce the perception of XOA value for corporate environment, hence reducing the product value.

So again, I prefer to tell that support is very important in production. You can still decide to NOT paying for it if you don't want to, but I don't want to miss opportunities because people will say "I'm doing production in my company but it's not really critical you know".

Jarli01 committed:
Update support.md

olivierlambert commented on docs/support.md:
```diff
@@ -22,7 +22,7 @@ If you are using Xen Orchestra via the source and not XOA, you can ask questions
 We encourage you to give back to the community by assisting other users via these two avenues as well.
 
-Lastly while Xen Orchestra is free and Open Source software, supporting and developing it takes a lot of effort. If you consider professional support business critical, please sign-up for one of our [support plans](https://xen-orchestra.com/#!/xo-pricing).
+Lastly while Xen Orchestra is free and Open Source software, supporting and developing it takes a lot of effort. If you are considering using Xen Orchestra in production, please subscribe for on of our [professional support plans](https://xen-orchestra.com/#!/xo-pricing). 
```
Trailing space

Jarli01 replied:
gah. . . fixed.

Jarli01 committed:
Update support.md
